### ISSUE TITLE
rgba(0,0,0, 1.0) throws error

### ISSUE THREAD

**Issue by viddo** [christian-bromann/rgb2hex#13]
```
> rgb2hex('rgba(0,0,0, 1.0)')
Error: given color (rgba(0,0,0, 1.0)) isn't a valid rgb or rgba color
    at rgb2hex (/tmp/repro/rgb2hex/index.js:35:15)
```

But alpha channel <1 or ===1 works fine:
```
> rgb2hex('rgba(0,0,0, 1)')
{ hex: '#000000', alpha: 1 }
> rgb2hex('rgba(0,0,0, 0.9)')
{ hex: '#000000', alpha: 0.9 }
```

**Comment by geosogel** [comment]
It can be solved by adding  |1.0* in the regular expression for parsing the input:
var digits = /(.*?)rgb(a)??\((\d{1,3}),(\d{1,3}),(\d{1,3})(,([01]|1.0*|0??\.([0-9]{0,})))??\)/.exec(strippedColor);
with 
ORIGINAL LINE: 
var digits = /(.*?)rgb(a)??\((\d{1,3}),(\d{1,3}),(\d{1,3})(,([01]|0??\.([0-9]{0,})))??\)/.exec(strippedColor); 

Pull request: #22

**Comment by christian-bromann** [comment]
Thanks @geosogel for providing a fix. Released as `v0.2.0`.
